mappings/net/minecraft/client/MinecraftClient.mapping

```diff
@@ -30,6 +30,7 @@ CLASS net/minecraft/class_1321 net/minecraft/client/MinecraftClient
 	FIELD field_26842 musicTracker Lnet/minecraft/class_6104;
 	FIELD field_26844 sessionService Lcom/mojang/authlib/minecraft/MinecraftSessionService;
 	FIELD field_26845 skinProvider Lnet/minecraft/class_6048;
+	FIELD field_26847 mainThread Ljava/lang/Thread;
```

Comment:
clientThread

Reply:
Upstream calls it just `thread`

Reply:
Since u would call it something like `this.client.thread` it should be obvious that its the client thread